src/toasts.ts

```diff
@@ -132,7 +132,7 @@ export class Toast {
   static _onDragStart(e: TouchEvent | MouseEvent) {
     if (e.target && (<HTMLElement>e.target).closest('.toast')) {
       const toastElem = (<HTMLElement>e.target).closest('.toast');
-      const toast: Toast = (toastElem as any).M_Toast;
+      const toast: Toast = toastElem['M_Toast'];
```

Comment:
I think we might need to revert this, since it introduces backwards compatibility issues with applications using the previous object orientated approach, propose to just set it to warnings until we can declare the correct type definition